src/components/Form.js

```diff
@@ -172,8 +173,19 @@ class Form extends React.Component {
     validate(values) {
         FormActions.setErrors(this.props.formID, null);
         FormActions.setErrorFields(this.props.formID, null);
+
+        // Run any validations passed as a prop
         const validationErrors = this.props.validate(values);
 
+        // Validate the input for html tags. It should superseed any other error
```

Comment:
```suggestion
        // Validate the input for html tags. It should supercede any other error
```

Superseed is what I use to get the grass on my lawn going!